distage-example-monomorphic-cats/src/main/scala/leaderboard/sql/SQL.scala

```diff
@@ -0,0 +1,22 @@
+package leaderboard.sql
+
+import cats.effect.IO
+import doobie.free.connection.ConnectionIO
+import doobie.util.transactor.Transactor
+import leaderboard.model.QueryFailure
+
+trait SQL {
+  def execute[A](queryName: String)(conn: ConnectionIO[A]): IO[Either[QueryFailure, A]]
+}
+
+object SQL {
+  final class Impl(
+    transactor: Transactor[IO[_]]
```

Comment:
Please remove redundant `[_]`'s in code.
```suggestion
    transactor: Transactor[IO]
```